wayfinder/README.md

```diff
@@ -0,0 +1,118 @@
+# Agent Check: Wayfinder
+
+## Overview
+
+[Wayfinder][1] is an infrastructure management platform that enables developer
+self-service through a centralized configuration. This check monitors Wayfinder
+key management components through the Datadog Agent.
+
```

Comment:
Can you please add more detail here on what this integration submits to Datadog? i.e if it's sending in metrics, and a summary of how those metrics bring value?

Reply:
updated